inc/tag.class.php

```diff
@@ -149,6 +154,14 @@ public static function install(Migration $migration) {
          }
       }
 
+      // Add rights. Default to full rights to maintain existing functionality.
+      $migration->addRight(self::$rightname);
```

Comment:
This line is not giving all rights to everyone, unlike information fiven in code comment. To give full rights to everyone, you have to change to:
```php
$migration->addRight(self::$rightname, ALLSTANDARDRIGHT, []);
```

But I am not sure this is what we want, as it will also affect all future installation. I propose to keep current version (and update comment), and let admins configure their profiles on plugin install/update.  We can add a comment in changelog to inform them that ACL have to be updated after installation of this new version.